benchmark/main.cpp

```diff
@@ -309,7 +309,10 @@ static int benchmark( int argc, char *argv[] )
 #if defined(MBEDTLS_MEMORY_BUFFER_ALLOC_C)
     unsigned char malloc_buf[HEAP_SIZE] = { 0 };
 #endif
-
+    // The call below is used to avoid the "unused parameter" warning.
+    // The context itself can be used by cryptographic calls which require it.
```

Comment:
An application like this test will not directly call a cryptographic function. The cryptographic functions should have a way of accessing the context like a ```getter``` as suggested in https://github.com/ARMmbed/mbedtls/issues/1200